src/content/docs/full-stack-observability/observe-everything/get-started/get-started-auto-telemetry-pixie

```diff
@@ -0,0 +1,88 @@
+---
+title: Get started with Auto-Telemetry with Pixie
+tags:
+  - Pixie Auto-Telemetry
+  - Agentless APM
+translate:
+  - jp
+metaDescription: 
+redirects:
+  - /docs/full-stack-observability/get-started-auto-telemetry-pixie
+---
+
+## Introducting Auto-Telemetry with Pixie
+
+<Callout variant="important">
+This is a beta release. As such... <add beta messaging>
+</Callout>
+
+With the launch of this beta, New Relic Kubernetes customers now have an alternative to manual instrumentation of services on their clusters, using Pixie Cloud. Pixie collects telemetry data using a virtual machine-like construct in the Linux kernel, eBPF. eBPF makes it easy to collect baseline data about their clusters without needing to manually instrument, change code, configure GUIs, or move data off clusters, like we see with traditional New Relic agents.
```

Comment:
The integration is sending data to New Relic, so we should remove 'move data off clusters'.

Reply:
fixed